akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/internal/PostgresOffsetStoreDao.scala

```diff
@@ -42,13 +61,16 @@ private[projection] class PostgresOffsetStoreDao(
     projectionId: ProjectionId)
     extends OffsetStoreDao {
 
-  private val logger = LoggerFactory.getLogger(getClass)
+  protected def logger: Logger = PostgresOffsetStoreDao.log
```

Comment:
why do we need to change this from val to def in the first place? doesn't `getLogger(getClass)` do exactly what we want? if that is the case we don't need the extra val in the companion object

Reply:
IIRC, in a-p-re2dc, I had to add it to avoid some conflicts caused by inheritance. Removing it here, maybe it was needed at some point, but is not anymore.